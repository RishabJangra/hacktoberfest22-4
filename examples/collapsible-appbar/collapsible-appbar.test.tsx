import React from 'react';
import renderer from 'react-test-renderer';

import { CollapsibleAppbarScreen } from './CollapsibleAppbar';

describe('Collapsible Appbar Tests', () => {
    it('Screen Renders', () => {
        const tree = renderer.create(<CollapsibleAppbarScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
