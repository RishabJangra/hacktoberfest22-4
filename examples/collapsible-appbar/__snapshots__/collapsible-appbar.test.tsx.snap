// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapsible Appbar Tests renders the screen 1`] = `
<RCTSafeAreaView
  accessible={true}
  emulateUnlessSupported={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#6200ee",
      "elevation": 0,
      "height": 420,
      "shadowColor": "rgba(0, 0, 0, 0.3)",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 2,
      "width": "100%",
    }
  }
>
  <Image
    resizeMethod="resize"
    source={1}
    style={
      Object {
        "height": 420,
        "opacity": 0.3,
        "position": "absolute",
        "resizeMode": "cover",
        "width": "100%",
      }
    }
    testID="header-background-image"
  />
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "minHeight": 112,
        "paddingBottom": 28,
        "paddingHorizontal": 16,
        "paddingVertical": 32,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 80,
          "margin": -16,
          "marginRight": 24,
          "opacity": 1,
          "padding": 16,
          "width": 80,
        }
      }
      testID="header-navigation"
    >
      <Text />
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "flex-end",
          "marginRight": 0,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 0,
            "justifyContent": "center",
          }
        }
      >
        <Text
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "white",
              "fontFamily": "System",
              "fontSize": 30,
              "lineHeight": 30,
              "textAlign": "auto",
              "writingDirection": "ltr",
            }
          }
          testID="header-title"
        >
          Collapsible Appbar
        </Text>
        <Text
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "white",
              "fontFamily": "System",
              "fontSize": 20,
              "lineHeight": 21,
              "opacity": 1,
              "textAlign": "auto",
              "writingDirection": "ltr",
            }
          }
          testID="header-info"
        >
          I'm hidden when collapsed
        </Text>
        <Text
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "color": "white",
              "fontFamily": "System",
              "fontSize": 18,
              "lineHeight": 18,
              "textAlign": "auto",
              "writingDirection": "ltr",
            }
          }
          testID="header-subtitle"
        >
          Tap to collapse/expand
        </Text>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "height": 112,
            "position": "absolute",
            "right": 8,
          },
          undefined,
        ]
      }
    />
  </View>
</RCTSafeAreaView>
`;
